listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx

```diff
@@ -1,25 +1,38 @@
 import * as React from "react";
 import Card from "../components/Card";
+import GlobalAppContext from "../utils/GlobalAppContext";
 
 export type ListenCountCardProps = {
+  isCurrentUser: Boolean;
```

Comment:
This prop isn't required, considering you user the `user` prop and the currentUser from the global context